app/js/views/services-list.jsx

```diff
@@ -21,4 +25,13 @@ export default class ServicesList extends React.Component {
       <ul className="service-list">{serviceNodes}</ul>
     );
   }
+
+  isUserFacingService(service) {
```

Comment:
This method should be moved to the `foxbox` lib and be called under the hood when you call `foxbox.getServices()`.